app.py

```diff
@@ -91,9 +91,7 @@ def register():
             mysql.connection.commit()
             msg = 'Usuario registrado exitosamente'
             print(fechaCreacion)
-    elif request.method == 'POST':
```

Comment:
As long as I understood this snippet would check if the method was POST but no field was filled.  If the `elif` is omitted, both POST and GET requests would fall into the  same condition of returning a view.

I'd keep this snippet as before.